subprojects/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -727,6 +743,18 @@ public List<CommandLineArgumentProvider> getJvmArgumentProviders() {
         return javaExecSpec.getJvmArgumentProviders();
     }
 
+    /**
+     * Returns the list of configured extra JVM arguments.
+     *
+     * @return A property of extra JVM arguments.
+     * @since 8.1
+     */
+    @Input
+    @Incubating
```

Comment:
I'm not sure what to do best - add this public api change to `accepted-public-api-changes` or mark in as a `Incubating`

Reply:
Definitely mark incubating